src/tools/rbd_mirror/Throttler.cc

```diff
@@ -0,0 +1,204 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include "Throttler.h"
```

Comment:
Nit: why the name change to something so generic? This seems to be heavily tied to sync throttling.